index.html

```diff
@@ -150,10 +208,62 @@ <h2>Methods</h2>
       <section>
         <h2><dfn>resetLatency</dfn> method</h2>
         <p>This method resets the latency counters. Note that it does not remove
-          latency information that has accrued but not yet been exposed through the API.</p>
+          latency information that has accrued but not yet been exposed through the API.
+        </p>
+        <p>
+          When <code>resetLatency()</code> is called, run the [= reset latency stats =] steps.
+        </p>
       </section>
     </section>
+
+
+    <section>
+    <h2>Internal algorithms</h2>
+
+    <p>
+    Running the <dfn>initialize playout stats</dfn> algorithm means running these steps:
+    <ol>
+      <li>Let <var>owningAudioContext</var> be the {{AudioContext}} passed into this algorithm.</li>
+      <li>Let <var>playoutStats</var> be a new instance of {{AudioPlayoutStats}}.</li>
+      <li>Set {{AudioPlayoutStats/[[audio context]]}} in <var>playoutStats</var> to <var>owningAudioContext</var>.</li>
+      <li>Run the [= reset latency stats =] steps.</li>
+      <li>Return <var>playoutStats</var>.</li>
+    </ol>
+  </p>
+
+<p>
+  Running the <dfn>reset latency stats</dfn> algorithm on an {{AudioPlayoutStats}} means running these steps: 
+  <ol>
+      <li>Let <var>currentLatency</var> be the playout latency of the last frame played by {{AudioPlayoutStats/[[audio context]]}}, or 0 if no frames have been played out yet.</li>
+      <li>Set {{AudioPlayoutStats/[[latency reset time]]}} to the current time.</li>
+      <li>Set {{AudioPlayoutStats/[[average latency]]}} to <var>currentLatency</var>.</li>
+      <li>Set {{AudioPlayoutStats/[[minimum latency]]}} to <var>currentLatency</var>.</li>
+      <li>Set {{AudioPlayoutStats/[[maximum latency]]}} to <var>currentLatency</var>.</li>
+    </ol>
+</p>
+
+<p>
+  Running the <dfn>update playout stats</dfn> algorithm on an {{AudioPlayoutStats}} means running these steps: 
+    <ol>
+    <li>If {{AudioPlayoutStats/[[audio context]]}} is not running, abort these steps.</li>
+    <li>Let <var>document</var> be the current object's <a href="https://html.spec.whatwg.org/multipage/nav-history-apis.html#concept-document-window">associated document</a>.</li> 
+    <li>Let <var>canUpdate</var> be false.</li> 
+    <li>If <var>document</var> is <a href="https://html.spec.whatwg.org/multipage/document-sequences.html#fully-active">fully active</a> and <a href="https://html.spec.whatwg.org/multipage/interaction.html#page-visibility">visible</a>, set <var>canUpdate</var> to true.</li>
+    <li>Let <var>permission</var> be the [=permission state=] for the permission associated with microphone access. 
```

Comment:
Check if `[="microphone"=]` works as a reference for the microphone permission.
Otherwise use https://w3c.github.io/mediacapture-main/#dfn-microphone

Reply:
It works, done